library/src/main/java/com/heinrichreimersoftware/materialintro/app/IntroActivity.java

```diff
@@ -429,12 +429,12 @@ public boolean goToSlide(int position) {
 
         if (position > lastPosition) {
             // Go forward
-            while (newPosition < position && canGoForward(newPosition, true)) {
+            while (newPosition < position && (canGoForward(newPosition, true) || forceScroll)) {
```

Comment:
The `forceScroll` should not "pollute" the position increasing/decreasing logic.
I would rather suggest having a simple if-statement like this:
```java
// ...
int newPosition;
if (forceScroll) {
    newPosition = position;
}
else {
    newPosition = lastPosition;
    // ... (position increasing/decreasing logic)
}

boolean blocked = false;
// ...
```